src/components/MentorsList/MentorsList.js

```diff
@@ -60,6 +62,7 @@ const MentorsList = props => {
         loadMore={loadMore}
         hasMore={mentorsInList.length < mentors.length}
       >
+        <Profile />
```

Comment:
This component should be in the new member area, home component below the `<Avatar />` component:

https://github.com/Coding-Coach/find-a-mentor/blob/db1209b33d5de29e018d23f316ffde31211ced09/src/Me/Routes/Home.js#L8